converter/internal/otelcolconvert/testdata/memorylimiter.yaml

```diff
@@ -0,0 +1,35 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+
+exporters:
+  otlp:
+    # Our defaults have drifted from upstream, so we explicitly set our
+    # defaults below (balancer_name and queue_size).
+    endpoint: database:4317
+    balancer_name: pick_first
+    sending_queue:
+      queue_size: 5000
+
+processors:
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 147
```

Comment:
@tpaschalis Are these out of sync with upstream? Can you create an issue to resync our defaults similar to the other ones I created?

Reply:
NVM I see that they are not the defaults of upstream

Reply:
No, I just added them to make sure that everything was going through properly. I'll remove them.

Reply:
I think it's fine to keep them in as a good test, I was just thrown off :)